toolhub_auth/forms.py

```diff
@@ -0,0 +1,70 @@
+from crispy_forms.layout import Layout, Fieldset, Submit, Field, Div, HTML
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import (
+    AuthenticationForm as DjangoAuthenticationForm,
+    # UserCreationForm as DjangoUserCreationForm,
+)
+from django.utils.translation import ugettext_lazy as _
+from utils.forms import CrispyFormMixin, FormActions
+
+
+class AuthenticationForm(CrispyFormMixin, DjangoAuthenticationForm):
+
+    has_columns = False
+
+    def __init__(self, *args, **kwargs):
+        super(AuthenticationForm, self).__init__(*args, **kwargs)
+        self.helper.layout = Layout(
+            Field("username"),
+            Field("password"),
+            Div(
+                Div(
+                    Submit(None, "Login"),
+                    HTML(
+                        '<a class="btn btn-secondary ml-auto" href="{% url \'password_reset\' %}">Forgot Password?</a>'
+                    ),
+                    css_class="d-flex",
+                ),
+                css_class="form-group mb-0",
+            ),
+        )
+
+
+class SignupForm(CrispyFormMixin, forms.ModelForm):
+    first_name = forms.CharField(required=True)
+    last_name = forms.CharField(required=True)
+
+    has_columns = False
+
+    class Meta:
+        model = get_user_model()
+        fields = ("email", "first_name", "last_name")
```

Comment:
Signup is broken because password1 isn't a field on the model and isn't included here on the `fields`. This also results in creating a user, with access to the site who doesn't have an email set.

Do we really want to just people signing up and not going through approval process of something? Seems like an invite email or link in slack when they become a member would be better way to manage new users.

Reply:
I agree that the signup process needs more work to verify the user. This is just to get things rolling. We can add an in to discuss and implement the final signup flow.